### PR TITLE
Package new GUI as RPM and docker image

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1603)

### CODE REVIEW

File: chroma-manager.conf.template [hunk 1]
```diff
@@ -78,7 +86,7 @@ server {
     }
 
     location ~ ^/$ {
-        return 301 https://$http_host/ui;
+        return 301 https://$http_host/ui2;
```
Comment (utopiabound) [inline chroma-manager.conf.template]: Should this be 302 until it can be changed back to `/ui`?

File: docker/iml-gui.dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+    apt-get install -y yarn nodejs && \
+    rustup target add wasm32-unknown-unknown && \
+    cargo install wasm-pack && \
+    yarn install && \
+    yarn build:release;
+
+FROM ubuntu
```
Comment (johnsonw) [inline docker/iml-gui.dockerfile]: Can this not be centos?

File: chroma-manager.conf.template (hunk 1 above)
Comment (jgrund) [reply chroma-manager.conf.template]: Maybe 307?

Comment (jgrund) [inline chroma-manager.conf.template]:
```suggestion
        return 307 https://$http_host/ui2;
```

Comment (jgrund) [reply chroma-manager.conf.template]: Updated to 307

File: docker/iml-gui.dockerfile (hunk 2 above)
Comment (jgrund) [reply docker/iml-gui.dockerfile]: It doesn't matter, we don't use it in production. The files are copied out of this container and into the nginx one.

Comment (johnsonw) [reply docker/iml-gui.dockerfile]: Ok, just wanted to make sure we don't introduce a new image if we can avoid it.

**Pull request merged**